corehq/apps/cloudcare/views.py

```diff
@@ -421,6 +423,23 @@ def get(self, request, *args, **kwargs):
         })
 
 
+def get_apps_by_user(domain, user_ids):
+    groups_by_user = get_groups_by_user(domain, user_ids)
+
+    app_access = ApplicationAccess.get_by_domain(domain)
```

Comment:
> the user has its group memberships

Oh? I don't think currently (i.e. outside of this PR) `LoginAsUsers` is including group info

```python
    def _format_user(self, user_json):
        user = CouchUser.wrap_correctly(user_json)
        return {
            'username': user.raw_username,
            'customFields': user.user_data,
            'first_name': user.first_name,
            'last_name': user.last_name,
            'phoneNumbers': user.phone_numbers,
            'user_id': user.user_id,
            'location': user.sql_location.to_json() if user.sql_location else None,
        }
```

Are you suggesting I include group info here, instead of adding alvailable app info to the user?